trunk/source/BA_SiteRepository/Ultimate_E4.32_Java21.target

```diff
@@ -5,11 +5,6 @@
 		<vmArgs>-ea -Xms4M -Xmx4G</vmArgs>
 	</launcherArgs>
 	<locations>
-		<location includeAllPlatforms="true" includeConfigurePhase="true" includeSource="false" type="InstallableUnit">
-			<repository id="ultimate-dependencies" location="https://struebli.informatik.uni-freiburg.de/p2-repository-ultimate-dependencies/"/>
-			<unit id="com.lmax.disruptor" version="3.3.5"/>
-			<unit id="org.yaml.snakeyaml" version="2.2.0"/>
-		</location>
```

Comment:
Can you leave the location but without units? So we still remember its there, just in case :)

Reply:
If we plan to use it later on (for JARs that are not available on Maven), this makes sense 😉